src/WebJobs.Extensions.DocumentDB/Bindings/CompositeBindingProvider.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Host.Bindings
+{
+    // TODO: Revisit this pattern to remove the need for this provider.
+    internal class CompositeBindingProvider : IBindingProvider
+    {
```

Comment:
Copying my other pattern for beta was faster than coming up with something new -- but now we have at least two bindings doing the same thing. Already logged as #48